packages/browser/src/tracing/request.ts

```diff
@@ -419,21 +413,9 @@ export function xhrCallback(
 }
 
 function addTracingHeadersToXhrRequest(xhr: SentryWrappedXMLHttpRequest, client: Client, span?: Span): void {
-  const scope = getCurrentScope();
-  const isolationScope = getIsolationScope();
-  const { traceId, spanId, sampled, dsc } = {
-    ...isolationScope.getPropagationContext(),
-    ...scope.getPropagationContext(),
-  };
-
-  const sentryTraceHeader =
-    span && hasTracingEnabled() ? spanToTraceHeader(span) : generateSentryTraceHeader(traceId, spanId, sampled);
```

Comment:
Here, we have checked for `span && hasTracingEnabled()`, while in fetch we did not check for `hasTracingEnabled()`. I opted to use this logic (just check span) everywhere now...